### PR TITLE
vundo.el: Don't show vundo commands in M-x (Emacs 28+)

### PR DESCRIPTION
**Pull request by condy0919**

### CODE REVIEW

**Comment by condy0919** [comment]
The copyright assignment is not required for 15 LOC changes.

File: vundo.el [hunk 1]
```diff
@@ -597,6 +597,7 @@ WINDOW is the window that was/is displaying the vundo buffer."
 
 (define-derived-mode vundo-mode special-mode
   "Vundo" "Mode for displaying the undo tree."
+  :interactive nil
```
Comment (ideasman42) [inline vundo.el:600]: A short comment explaining why interactive is `nil` would be good.

```
;; This minor mode only applies to buffers created by `vundo',
;; there is no need to show this minor mode to the user.
```

Comment (condy0919) [reply vundo.el:600]: Great, I will take yours.

**Comment by casouri** [comment]
Thanks, merged.

**Comment by casouri** [comment]
This is very close to 15 LOC so I suggest getting an assignment, maybe there will be more contributions in the future :-)

**Pull request closed without merging**